src/NEO/Address.h

```diff
@@ -13,25 +13,31 @@
 #include <string>
 
 namespace TW::NEO {
+    static const unsigned AddressSize{21};
 
-class Address : public TW::Base58Address<21> {
```

Comment:
can you follow existing code styles? no indentation for class definition, and two spaces for `public`

Reply:
I'm removing the indentation for all classes, when you say two spaces for `public`, do you mean for all kinds of indentation?